src/pyfbad/data/database.py

```diff
@@ -237,21 +245,136 @@ def create_mysql_conn(self):
+        """
+        try:
+            print("Writing data to {0}...".format(table_name))
+            data.to_sql(name=table_name, con=conn_mysql, index=_index)
+        except Exception as e:
+            print(e.args)
+            sys.exit(2)
+
+        finally:
+            conn_mysql.close()
+
+
+class PostgreSQLDB:
+
+    def __init__(self, POSTGRESQL_USERNAME, POSTGRESQL_PASSWORD, POSTGRESQL_HOST, POSTGRESQL_PORT, POSTGRESQL_DATABASE):
+        """ Get the connection configuration for postgresql.
+        Args:
+            POSTGRESQL_USERNAME (str): Database username.
+            POSTGRESQL_PASSWORD (str): Databse password
+            POSTGRESQL_HOST (str): Database host name
+            POSTGRESQL_PORT (str): Database port name
+            POSTGRESQL_DATABASE (str): Database name
+        Returns: None
+        """
+        self.POSTGRESQL_USERNAME = POSTGRESQL_USERNAME
+        self.POSTGRESQL_PASSWORD = POSTGRESQL_PASSWORD
+        self.POSTGRESQL_HOST = POSTGRESQL_HOST
+        self.POSTGRESQL_PORT = POSTGRESQL_PORT
+        self.POSTGRESQL_DATABASE = POSTGRESQL_DATABASE
+
+    def set_postgredb_conn(self):
+        """ Set a connection configuration for postgredb.
+        Args: None
+        Returns: None
+        """
+
+        try:
+            print("Setting postgredb connection parameters...")
+
+            self.postgresql_connection_string = 'postgresql://{0}:{1}@{2}:{3}/{4}'.format(
+                self.POSTGRESQL_USERNAME,
+                self.POSTGRESQL_PASSWORD,
+                self.POSTGRESQL_HOST,
+                self.POSTGRESQL_PORT,
+                self.POSTGRESQL_DATABASE)
+
+        except:
+            print("Something went wrong when setting postgredb connection parameters.")
+
+    def create_postgredb_conn(self):
+        """ Create database engine to connect postgresql database.
+        Args: None
+        Returns:
+            engine (Database instance): Engine instance
+        """
+        try:
+            print("Creating postgredb connection...")
+            engine = create_engine(self.postgresql_connection_string)
+            return engine.connect()
+
+        except:
+            print("Something went wrong when creating postgredb connection.")
+
+    def reading_rawdata(self, query, db_conn, table_name):
+        """ Reading row data from postgredb with sql query.
+        Args: 
+            query (str): Database username
+            db_conn (Database instance): Engine instance
+            table_name (str): postgredb table name for dataframe
+        Returns: data (DataFrame): A dataframe ingested with sql query
+        """
+        try:
+            print("Reading data from {0}...".format(table_name))
+            return pd.read_sql_query(text(query), db_conn)
+
+        except Exception as e:
+            print(e.args)
+            sys.exit(3)
+
+        finally:
+            db_conn.close()
+
+    def writing_rawdata(self, data, db_conn, table_name, _index=False):
```

Comment:
please add the chunk size parameter to this method and to_sql function, default value could be 10000